hackathon_site/dashboard/frontend/src/pages/Teams/Teams.test.tsx

```diff
@@ -1,10 +1,13 @@
 import React from "react";
 
 import { render } from "testing/utils";
-
+import { teamsList } from "testing/mockData";
 import Teams from "pages/Teams/Teams";
 
-test("renders without crashing", () => {
+test("renders teams without crashing", () => {
+    const teamCodes = teamsList.map((team) => team.TeamName);
     const { getByText } = render(<Teams />);
-    expect(getByText("IEEEEEE")).toBeInTheDocument();
+    teamCodes.forEach(function (teamCode) {
```

Comment:
arrow function here as welllll

Reply:
also, minor thing but you don't need to do
`const teamCodes = teamsList.map((team) => team.TeamName);`
you can do
`teamsList.forEach(({ teamCode }) => { ...})` the curly brackets will extract the teamCode property from the objects in teamList

Reply:
turned into arrow function!